OpenUpSA/wazimap-ng-ui
# Show subcategories if key metrics are present in rich data view

## Description
Show subcategory if key metrics are available even if:
* There are no indicators
* All indicators are hidden

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-706

## How is it tested automatically?
Added unit tests for subcategory visibility in rich data view

## How should a reviewer test it locally
Make sure we have key metrics in subcategory
* Hide all indicators in subcategory but subcategory should not be set as hidden
* even if there are no indicators present in subcategory still subcategory should be visible

## Screenshots


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
